src/content/docs/more-integrations/cloudfoundry-integrations/vmware-tanzu-dotnet-buildpack-integration.mdx

```diff
@@ -0,0 +1,131 @@
+        </td>
+
+        <td>
+            1.1.12
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Release date
+        </td>
+        <td>
+            August 15, 2023
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Software component version
+        </td>
+        <td>
+            New Relic Dotnet Extension Buildpack 1.1.12
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Compatible Ops Manager version(s)
+        </td>
+        <td>
+            2.9.x, 2.10.x, and 3.0.x
+        </td>
+    </tr>
+    <tr>
+        <td>
+            Compatible VMware Tanzu Application Service for VMs versions
+        </td>
+        <td>
+            2.10.x, 2.11.x, 2.12.x, 2.13.x, 3.0.x, and 4.0.x
+        </td>
+    </tr>
+    <tr>
+        <td>
+            BOSH stemcell version
+        </td>
+        <td>
+            Ubuntu Jammy
+        </td>
+    </tr>
+    <tr>
+        <td>
+            IaaS support
+        </td>
+
+        <td>
+            AWS, GCP, Azure, and vSphere
+        </td>
+    </tr>
+    </tbody>
+</table>
+
+## Install the buildpack [#install]
+
+The New Relic Dotnet Extension Buildpack for VMware Tanzu can be installed via the tile in Ops Manager. Alternatively, you can extract the `.Pivotal` file and install individual extension buildpacks using the cf Command Line Interface (CLI) command `cf create-buildpack`.
+
+After you start monitoring your apps, you can set alerts based on any metrics that are collected by Dotnet agents using the New Relic alerting subsystem.
+
+The New Relic Dotnet Extension Buildpack for VMware Tanzu installs one or more of the following buildpacks depending on the tile configuration (total of 8 extension buildpacks):
+
+* New Relic Dotnet Core Extension Buildpack for Dotnet Core Applications running on Ubuntu Jammy 1.* stacks. This extension buildpack is non-cached.
+* New Relic Dotnet Core Extension Cached Buildpack for Dotnet Core Applications running on Ubuntu Jammy 1.* in disconnected (isolated) VMware Tanzu deployments. This is for support of air-gapped environments where there is no access to the outside world.
+* 3 New Relic HWC Extension Buildpacks for Dotnet Framework Applications running on Windows 2019 stacks. This extension buildpack is non-cached.
+* 3 New Relic HWC Extension Cached Buildpacks for Dotnet Framework Applications running on Windows 2019 stacks in disconnected (isolated) VMware Tanzu deployments. This is for support of air-gapped environments where there is no access to the outside world.
+
+All buildpacks use the multi-buildpack approach of Cloud Foundry and require either the standard Dotnet Core buildpack or HWC buildpack to be specified as the last buildpack in the buildpack chain, either in the app manifest or in the `cf push` command line.
+
+<Callout variant="important">
+    The cached version of this extension buildpack for both Dotnet Core and Dotnet Framework contains New Relic Dotnet Agents version `9.1.1`
```

Comment:
Does this mean that this buildpack is only compatible with this agent version? We should x-check what our current supported DOTNET agent is.

Reply:
No, compatibility is taken care in the integration itself.